build/configs/rtl8720e/rtl8720e_download.sh

```diff
@@ -30,7 +30,7 @@ function pre_download()
 {
 	source ${TOP_PATH}/os/.bininfo
 	cp -p ${BIN_PATH}/km4_boot_all.bin ${IMG_TOOL_PATH}/km4_boot_all.bin
-	cp -p ${BIN_PATH}/kernel_rtl8720e_200204.trpk ${IMG_TOOL_PATH}/kernel_rtl8720e_200204.trpk
+	cp -p ${BIN_PATH}/${KERNEL_BIN_NAME} ${IMG_TOOL_PATH}/${KERNEL_BIN_NAME}
```

Comment:
Could you please add explain in commit?

I'm aware as the reason need to changed this is setting to fiilname that include version in os/toos/set_bininfo.py

Reply:
@Tirtha-Samsung Could you leave some description on the commit description?